tests/e2e/CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Change Log
 
-All notable changes to this project will be documented in this file. See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# [4.3.0](https://github.com/algolia/instantsearch/compare/@instantsearch/e2e-tests@4.0.0...@instantsearch/e2e-tests@4.3.0) (2023-09-19)
+
+
+### Bug Fixes
+
+* **prettier:** consistent version ([#5850](https://github.com/algolia/instantsearch/issues/5850)) ([ca59c6d](https://github.com/algolia/instantsearch/commit/ca59c6dbd5c9eac4e2e0179a24e39bca997ae141))
+* **this:** ensure all functions are able to be destructured ([#5611](https://github.com/algolia/instantsearch/issues/5611)) ([a8b5c1e](https://github.com/algolia/instantsearch/commit/a8b5c1e5bbd6afac39fce523f7d7c2ec02f51153)), closes [#5589](https://github.com/algolia/instantsearch/issues/5589)
+
+
+### Features
+
+* rename packages ([#5787](https://github.com/algolia/instantsearch/issues/5787)) ([c133170](https://github.com/algolia/instantsearch/commit/c133170e563592dfc15a95daced1f8447327a09a))
```

Comment:
This is already part of 4.2.0, there must be an issue with how the changelog is generated.

```suggestion
```

Reply:
same, it's because we don't tag